src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -618,6 +618,49 @@ public KojiImportResult importBuild( KojiImport importInfo, Iterable<Supplier<Im
         }, "Failed to execute content-generator import" );
     }
 
+    public <T extends KojiQuery> List<Integer> queryCountOnly( String method, List<T> queries, KojiSessionInfo session )
```

Comment:
There is a single method for multicall, but I have separate methods for each non-multicall version.